amazon-kinesis-client/src/main/java/software/amazon/kinesis/metrics/MetricsCollectingTaskDecorator.java

```diff
@@ -52,6 +52,7 @@ public TaskResult call() {
         try {
             result = other.call();
         } finally {
+            MetricsUtil.addLatency(scope, other.getClass().getSimpleName(), startTimeMillis, MetricsLevel.SUMMARY);
```

Comment:
method signature:

```java
    public static void addLatency(@NonNull final MetricsScope metricsScope, final String dimension,
                                  final long startTime, @NonNull final MetricsLevel metricsLevel)
```

Reply:
Can you post the new metrics emitted with this change and their value?

Reply:
sure, will do